.gitignore

```diff
@@ -0,0 +1 @@
+/[Bb]uild/
```

Comment:
Don't think we need that.

Reply:
the folder name is defaulted by Gradle, so I'm undecided here.